packages/iTwinUI-react/src/core/Menu/Menu.tsx

```diff
@@ -102,7 +102,7 @@ export const Menu = React.forwardRef<HTMLUListElement, MenuProps>(
 
     return (
       <ul
-        className={cx('iui-menu', className)}
+        className={cx('iui-menu', 'iui-scroll', className)}
```

Comment:
I am not sure about this change because `Menu` is its own reusable component so if someone want to show very long menu or without scrollbar, they will have to write some css to undo what `iui-scroll` does. Also, when using Menu as e.g. context menu, for me scrollbar doesn't look nice. Maybe add a prop that adds this class and enable it for select, combobox and action column.

Reply:
Ohh, right, that makes sense, thanks. Actually during today's meeting, Mayank pointed out that I misunderstood the task a bit. The task for v1 was just to remove the inline styles for `maxHeight` since `max-height` was specified in `.iui-scroll`.

As a result, I just removed `.iui-scroll` from the `Menu` and added (undoing my removal of) `.iui-scroll` in the components that use `Menu` (`ComboBoxMenu`, etc.). I then made your idea a new enhancement issue #853.